codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -557,7 +709,8 @@ int incrementStreamWindow(int delta) throws Http2Exception {
                 throw streamError(stream.id(), FLOW_CONTROL_ERROR,
                         "Window size overflow for stream: %d", stream.id());
             }
-            int previouslyStreamable = streamableBytes();
+            final int previouslyStreamable = streamableBytes();
+            final int previousWritableWindow = writableWindow();
```

Comment:
I originally didn't want to introduce additional overhead...but this is simpler. Will update.